lib/blocks.php

```diff
@@ -16,6 +16,36 @@ function gutenberg_reregister_core_block_types() {
 		return;
 	}
 
+	$block_folders = array(
+		'audio',
+		'button',
+		'buttons',
+		'file',
+		'gallery',
+		'group',
+		'heading',
+		'html',
+		'image',
+		'list',
+		'media-text',
+		'missing',
+		'more',
+		'navigation-link',
+		'nextpage',
+		'paragraph',
+		'preformatted',
+		'pullquote',
+		'quote',
+		'separator',
+		'social-links',
+		'spacer',
+		'subhead',
+		'table',
+		'text-columns',
+		'verse',
+		'video',
+	);
```

Comment:
Miss blocks, 
- Code
- Classic
- Column
- Columns
- Cover
- Embed

Reply:
Good catch, I will print a list of existing blocks and check which I missed and which don't contain `block.json` (at least Embed misses this file).